lisa/tools/dmesg.py

```diff
@@ -82,6 +86,24 @@ def get_vmbus_version(self) -> VersionInfo:
                 return VersionInfo(int(major), int(minor))
         raise LisaException("No find matched vmbus version in dmesg")
 
+    def get_lis_version(self) -> VersionInfo:
```

Comment:
There is an Azure tool called `LisDriver` already in `lisa\sut_orchestrator\azure\tools.py`. Check if it's fit into this scenario. The LIS is Microsoft specific things, it's better not to put it in generic tool, dmesg.

Reply:
The modinfo tool also can get version of hv_vmbus.